OpenSim/Simulation/Model/ConditionalPathPoint.cpp

```diff
@@ -208,12 +138,9 @@ void ConditionalPathPoint::setRangeMin(const SimTK::State& s, double aMin)
  *
  * @param aRange range max to change to.
  */
-void ConditionalPathPoint::setRangeMax( const SimTK::State& s, double aMax)
+void ConditionalPathPoint::setRangeMax(double maxVal)
```

Comment:
`@param aRange` -> `@param maxVal` (or just delete the comment because it's obvious)